lib/app-service-registration.js

```diff
@@ -56,6 +57,22 @@ AppServiceRegistration.prototype.getId = function() {
 };
 
 /**
+ * Set the list of protocols that this appservice will serve for third party lookups.
+ * @param {Array.<String>} protocols The protocols
```

Comment:
I think the other JSDocs we have use `string[]` so better be consistent with that.